### ISSUE TITLE
Version Mismatch

### ISSUE THREAD

**Issue by kiswa** [aseprite/aseprite#1617]
After building and running Aseprite displays that it is v1.3-dev instead of v.1.2.4

[See here](https://github.com/aseprite/aseprite/blob/33507fd09772f1bc8a822092e3a5d8a17f8c6002/src/config.h#L30).

### Aseprite and System version

* Aseprite version: 1.2.4, [release source download](https://github.com/aseprite/aseprite/releases/download/v1.2.4/Aseprite-v1.2.4-Source.zip)
* System: Arch Linux

**Comment by dacap** [comment]
Yeah, actually this is a problem in my build pipeline. The first thing that I do is creating the `Source.zip` file from the Git content, then, I use that zip to build on each platform (the build process includes a script that modify the version). In a future I'll modify this so the version is updated before the compression.
